requirements-dev.txt

```diff
@@ -1,11 +1,11 @@
 -r requirements.txt
 babel
-django_extensions
 flake8
 ipython
 pickleshare
 Werkzeug
 django-debug-toolbar
+drf-extensions
```

Comment:
I think you wanted to keep 'django_extensions' here, and leave 'drf-extensions' in requirements.txt